manifest.php

```diff
@@ -41,7 +41,7 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '14.6.1',
+    'version' => '14.6.2',
```

Comment:
do we really need this? there is no change inside at all?

Reply:
Already pushed npm dependencies update